### ISSUE TITLE
As a special uses applicant, I want to know how long it generally takes to complete my application online so that I can manage my time.

### ISSUE THREAD

**Issue by ASprinkle** [USDAForestService/fs-open-forest-platform#530]
## Notes
-Applicants want to know how long it takes for them to fill out their application online (i.e., plan on no less than xx minutes/hours, progress bar, knows what material they need before starting, etc.).

## Acceptance Criteria
- [x] Applicants are informed of time expectations at the beginning and throughout the application process.

## Tasks
- [x] Determine whether or not time expectations can be communicated throughout the process, or only at the beginning of the process or at the beginning of each step.
- [x] Research time to task completion estimates for each step/application process.
- [x] Mock designs showing the time estimate in the Learn More sections of Non-Com and OG pages as a line of text under the Permit Information title.
- [x] Add the applicant time estimate to the Learn More sections of Non-Com and OG pages as a line of text under the Permit Information title.
- [x] PO approved.

## Definition of Done
- [x] Pull requests meet technical defintion of done
- [x] Compare finished design with mockup
- [x] Usability tested

**Comment by lisaredux** [comment]
This design shows this information on a breakout page from the SU permit landing <img width="322" alt="Screenshot 2019-03-13 10 26 18" src="https://user-images.githubusercontent.com/26801590/54288904-a0f33500-457e-11e9-92fa-5bf068e4f43a.png">

**Comment by ASprinkle** [comment]
That is pretty clear to me. It will probably take some iterations of going through the application process to figure out some kind of average time to complete application for each use type.

Also for consideration - the time estimate would be based on the user having all of their documents ready to go. The time estimate should also include the time it takes for the user to set up their authentication account (?) Interested on some thoughts there...

**Comment by lisaredux** [comment]
Needs: 
time estimate
progress concept

**Comment by carlsonem** [comment]
@ASprinkle This design is ready for your approval when you return to the team

**Comment by ASprinkle** [comment]
The design looks good to me.  I won't select "PO Approved" here as this is still in the backlog and  I would think that something may change along the way as we work on building it and testing in staging.

**Comment by MelissaBraxton** [comment]
@ASprinkle  - Great call. Let's move this back to the backlog so that it can be prioritized along with the other stories there.

**Comment by lisaredux** [comment]
@ASprinkle @MelissaBraxton 
this story still needs:
1) time estimates for non-comm and temporary outfitters (depends on authentication - login.gov or eAuth1) 
2) progress concept
3) where the content is going to go, like story #531 - on the same page or breakout page 

<img width="1015" alt="Screenshot 2019-04-12 16 56 51" src="https://user-images.githubusercontent.com/26801590/56136526-f6957580-5f60-11e9-984a-f79c63d39fa7.png">

**Comment by ASprinkle** [comment]
Need to update Tasks to reflect time estimates to manage applicant expectations for.
Time est: Authenticate (Create profile)
Time est: Noncommercial group use
Time est: Temp outfitter guide permit.

We're looking for a range to complete all tasks.  
For example when someone selects "Apply" for temp outfitter permit, they would be informed as to how much time they could expect it might take to fill out their permit, and submit it.

If applicants have a better understanding of how much time it takes for them to fill out the applications, they may be more likely to have a better user experience.

**Comment by jdamis4XD** [comment]
@lisaredux Amber added the new tasks and criterias.

**Comment by lisaredux** [comment]
> Need to update Tasks to reflect time estimates to manage applicant expectations for.
> Time est: Authenticate (Create profile)
> Time est: Noncommercial group use
> Time est: Temp outfitter guide permit.
> 
> We're looking for a range to complete all tasks.
> For example when someone selects "Apply" for temp outfitter permit, they would be informed as to how much time they could expect it might take to fill out their permit, and submit it.
> 
> If applicants have a better understanding of how much time it takes for them to fill out the applications, they may be more likely to have a better user experience.

Completely agree @ASprinkle . I'll start looking at the videos to parse out the times too. Do you know where those live?

Also, do you have capacity @ASprinkle this week for a meeting to discuss where this information will ultimately live?

**Comment by ASprinkle** [comment]
@lisaredux I do not know for sure where those videos live. I would have to check the Google Drive shared documents...

**Comment by lisaredux** [comment]
> @lisaredux I do not know for sure where those videos live. I would have to check the Google Drive shared documents...

@ASprinkle Thanks for checking. Turns out Anne has them off the shared drive because of PII and is going to make some available to us to view for time estimations (with a large range due to it being a usability study)

**Comment by lisaredux** [comment]
@ASprinkle — beginning to end times of our usability study videos with login.gov (with Anne's questions in there): 
Temp outfitter lengths: 40:01, 45:10, 48:28, 1:01:31, 1:04:06
Non-comm: 19:36, 21:13, 22:50, 23:13, 50:40

We will have some as examples to watch and get an idea of as well.

**Comment by annepetersen** [comment]
I'll also note that the non-comm participant who came in at 50:40 also took a quick run-through of the temp outfitter application in that time.

**Comment by annepetersen** [comment]
I’d say the initial setup/screenshare/etc. usually took about 5 minutes or so for each.

**Comment by ASprinkle** [comment]
@annepetersen and @lisaredux Thank you!.. Looks like about an average of 30 minutes for Non-Commercial, and about an hour for temporary outfitter guide permit application.

**Comment by lisaredux** [comment]
Possible suggested ranges:
Non-commercial- 20-30 minutes (with login.gov)
Temporary outfitters- 35-60 minutes (with login.gov)

**Comment by ASprinkle** [comment]
@lisaredux perhaps this story is also dependent on what we find out form the research spike #608

**Comment by lisaredux** [comment]
@ASprinkle @carlsonem @DawnDesigner - this is now in refining, and design will discuss design possibilities next week in person

**Comment by jdamis4XD** [comment]
@ASprinkle @carlsonem @DawnDesigner @lisaredux We will be meeting this week to discuss as we didn't have the chance last week.

**Comment by lisaredux** [comment]
@ASprinkle we (@jdamis4XD @DawnDesigner and I) are collectively working on this, but we will need to schedule a meeting with you to vet the new content organization. I am currently working on wireframing the content for this and #531 and #627.

**Comment by lisaredux** [comment]
Wireframe of the updated content flow under review by @ASprinkle  https://sketch.cloud/s/MGMV9

**Comment by ASprinkle** [comment]
@lisaredux same comments as #531 

It is clear to me that it may take X amount of time to complete each set of applications.

**Comment by DawnDesigner** [comment]
A/B testing with two prototypes in progress for these issues and will be removed from blocked when testing is completed and these issues are refined and put into ready.
#530 , #531, #565 and #627

**Comment by DawnDesigner** [comment]
This testing study is now complete. Findings and recommendations can be found on this repositories Wiki (Design and Research) here:

https://github.com/18F/fs-open-forest-platform/wiki/Usability-Testing:-Special-Uses-(Content-Flow-Study)-June-2019

**Comment by DawnDesigner** [comment]
This testing study is now complete. Findings and recommendations can be found on this repositories Wiki (Design and Research) here:

https://github.com/18F/fs-open-forest-platform/wiki/Usability-Testing:-Special-Uses-(Content-Flow-Study)-June-2019

**Comment by ASprinkle** [comment]
@lisaredux @DawnDesigner can we mock the time to complete the application language on the page to prepare it for dev?

**Comment by DawnDesigner** [comment]
@ASprinkle  @lisaredux Here is the mock up for #565 that shows the text and where it sits on the page.  the text is not yet finalized, but is acting as a placeholder.

![SU - Landing Page 3](https://user-images.githubusercontent.com/47989077/60106548-64190b80-9733-11e9-993b-6a451d35b09c.png)

**Comment by DawnDesigner** [comment]
Here is the final location of this information on the Non-Commercial and Outfitters details page.

![TimeTextLocation Noncommercial](https://user-images.githubusercontent.com/47989077/60185827-9ccee880-97f8-11e9-93a0-fc3293d7d184.PNG)
![TimeTextLocationOutfitter](https://user-images.githubusercontent.com/47989077/60185829-9ccee880-97f8-11e9-9c37-514b1ca26e41.PNG)

**Comment by ASprinkle** [comment]
Comments from #627 
Non-commercial permit - Change to: "Non-commercial Group Use Permit"
Temporary Outfitter's Permit - Change to: "Temporary Outfitter and Guide Permit"
Application Time: Appx. 35 minutes to 1 hour - Change to: "Application Time: Appx. 20-30 minutes"
Application Time: Appx. 1 to 3 hours - Change to" Application Time: Appx. 1 hour"

Please make the following text changes. I referenced back to 530, and would like to see 20-30 mins for Non-com and 1 hour for Temp OG. This is close to what was researched.

**Comment by lisaredux** [comment]
@ASprinkle here's the update, also in #627
![SpecialUse-LandingPage07032019](https://user-images.githubusercontent.com/26801590/60622228-bc907e80-9dad-11e9-8ab0-d57c164428a1.png)


This is ready for dev @mtlaney. Including a PDF to grab text content from. Let us know if you have any questions.
[SpecialUse-LandingPage07032019.pdf](https://github.com/18F/fs-open-forest-platform/files/3356696/SpecialUse-LandingPage07032019.pdf)

**Comment by lisamcmichael** [comment]
Met w/ Amber: Learn More page needs to be updated with the PO approved information; see June 26 entry

**Comment by lisamcmichael** [comment]
@lisaredux adding text to both Learn More pages to indicate placement on the page.

**Comment by lisamcmichael** [comment]
@ASprinkle I wonder if this can be set to another status .... when Lisa gets back she can add the text to the 2 mockups and then it's ready for Brian or Mike.

**Comment by lisamcmichael** [comment]
@lisaredux When you have a few, let me know if I can help get this story ready and over to dev.

**Comment by lisaredux** [comment]
@ASprinkle @lisamcmichael working on this story this afternoon. Update after our meeting, hopefully

**Comment by lisaredux** [comment]
@ASprinkle trying to find the best location for this timing information on the Learn More pages. And in order to do that, trying to figure out what happened to Application Requirements sections. Did we forget to include that on the Learn More pages? I'm having trouble remembering where we might have missed some content. Please advise.

> Here is the final location of this information on the Non-Commercial and Outfitters details page.
> 
> ![TimeTextLocation Noncommercial](https://user-images.githubusercontent.com/47989077/60185827-9ccee880-97f8-11e9-93a0-fc3293d7d184.PNG)
> ![TimeTextLocationOutfitter](https://user-images.githubusercontent.com/47989077/60185829-9ccee880-97f8-11e9-9c37-514b1ca26e41.PNG)


Currently:
<img width="919" alt="Screenshot 2019-11-14 11 31 35" src="https://user-images.githubusercontent.com/26801590/68876447-77247c00-06d2-11ea-8ccf-124b6d7e0285.png">

**Comment by ASprinkle** [comment]
@lisaredux  I see what you're talking about in terms of the content inconsistency from the old Mock to what is currently showing on the Learn More page. I do not recall where we may have made those changes. However when I look at that a little closer, and think about our "required fields" page, I do not think it is "required" to submit an "operating plan". We'll need to research this as a separate story.  In the meantime, can we mock up the design to meet the intent of this story. I created this document as a guide to help clarify.
[530 TimeExpectationLearnMore.pdf](https://github.com/USDAForestService/fs-open-forest-platform/files/3848128/530.TimeExpectationLearnMore.pdf)

**Comment by lisaredux** [comment]
@ASprinkle Thanks for clarifying. Oops, yeah — complete space out on the correct page yesterday. Here is the update....

Noncommercial Learn More page
https://open-forest-staging.app.cloud.gov/mbs/applications/noncommercial-group-use/learn-more

Add: "It may take approximately 20-30 minutes to complete this online application."

<img width="1062" alt="Screenshot 2019-11-15 11 11 59" src="https://user-images.githubusercontent.com/26801590/68957936-17dd6f00-0799-11ea-97d7-a845e76e89d5.png">



Temporary Outfitter's Learn More page
https://open-forest-staging.app.cloud.gov/mbs/applications/temp-outfitters/learn-more

Add: "It may take approximately 40-65 minutes to complete this online application."

<img width="1063" alt="Screenshot 2019-11-15 11 13 14" src="https://user-images.githubusercontent.com/26801590/68957945-1b70f600-0799-11ea-8bfa-f62cce3e1d4d.png">

**Comment by ASprinkle** [comment]
@lisaredux  Did you see the document I added in my comment above? ^

**Comment by lisaredux** [comment]
Thanks @ASprinkle— oops again. Hopefully 3rd times a charm....

Noncommercial Learn More page
https://open-forest-staging.app.cloud.gov/mbs/applications/noncommercial-group-use/learn-more

Add: "It may take approximately 20-30 minutes to complete this online application."

<img width="1095" alt="Screenshot 2019-11-15 11 50 00" src="https://user-images.githubusercontent.com/26801590/68961679-e9639200-07a0-11ea-805c-fac608cacb0a.png">


Temporary Outfitter's Learn More page
https://open-forest-staging.app.cloud.gov/mbs/applications/temp-outfitters/learn-more

Add: "It may take approximately 40-65 minutes to complete this online application."

<img width="1073" alt="Screenshot 2019-11-15 11 54 42" src="https://user-images.githubusercontent.com/26801590/68961745-0dbf6e80-07a1-11ea-91bf-0e4beadee576.png">

**Comment by ASprinkle** [comment]
@lisaredux  this looks great!! :)

**Comment by ASprinkle** [comment]
![image](https://user-images.githubusercontent.com/33435621/69986609-9bfe5900-14f2-11ea-985c-dd5402410b4f.png)
![image](https://user-images.githubusercontent.com/33435621/69986613-9e60b300-14f2-11ea-9082-94f29c6959e9.png)

**Comment by ASprinkle** [comment]
@mtlaney I reviewed this in prep to move to done.
In Staging: https://open-forest-staging.app.cloud.gov/mbs/applications/noncommercial-group-use/learn-more I see the update, and it looks good.
![image](https://user-images.githubusercontent.com/33435621/70182058-f684ea00-1697-11ea-9a14-38f57128e4c1.png)

However, when I go to the MBS special uses production site https://openforest.fs.usda.gov/mbs/applications/noncommercial-group-use/learn-more 
It does not look like the update took effect, any ideas why?
![image](https://user-images.githubusercontent.com/33435621/70182101-0ac8e700-1698-11ea-9d73-9eacc7fa2b36.png)

**Comment by mtlaney** [comment]
@ASprinkle I'm assuming it had to do with our deploy failure last night due to space limitations on the api-server. Abdul has corrected and new deployment successfully went thru at 11pm est tonight. If you can, check in the morning for the updated changes :)

**Comment by ASprinkle** [comment]
@mtlaney thanks!! it's looking great in Prod now!
